akka-sample-distributed-data-scala/src/multi-jvm/scala/sample/distributeddata/VotingServiceSpec.scala

```diff
@@ -62,8 +64,6 @@ class VotingServiceSpec extends MultiNodeSpec(VotingServiceSpec) with STMultiNod
     }
 
     "count votes correctly" in within(15.seconds) {
-      import VotingService._
-      val votingService = system.spawn(VotingService(), "votingService")
```

Comment:
as far as I can see it should work if it's started later, here, also

Reply:
Sorry, what do you mean?

Reply:
I mean, that this change shouldn't be necessary for making it work. If the dead letters we saw in travis disappear due to this change we still have something strange to investigate.

Reply:
Ah right.  I agree: I reverted it to make the port stay true to its original.